java/ingest/ingest-core/src/main/java/sleeper/ingest/job/status/IngestJobInfoStatus.java

```diff
@@ -18,6 +18,14 @@
 
 import sleeper.core.record.process.status.ProcessRunStartedUpdate;
 
+/**
+ * An interface for an ingest job process started update that has information about the input files.
```

Comment:
We can omit "An interface" here. See the Oracle style guide, "Class/interface/field descriptions can omit the subject and simply state the object": https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html